HISTORY.rst

```diff
@@ -3,6 +3,19 @@
 Release History
 ===============
 
+0.11.0
++++++++++++++++
+
+**IoT Central updates**
+
+* Adds preview commands (v1.1-preview).
+* Adds x-ms-client-request-id header for each request
+
+**Breaking Changes**
+
+* List commands like `az iot central device list` and others, 
```

Comment:
Can we be specific on which commands have breaking changes? Are these GA (non-preview) commands?

Reply:
added. some of them are GA, some preview. worth specifying in history?

Reply:
The history content looks much nicer now. Just formatting appears a bit off in the first list

![image](https://user-images.githubusercontent.com/28658112/133701247-ee181908-8ad1-4eb1-a862-361d32ca8fb6.png)

Breaking changes on GA commands are concerning though, has the impact been looked into? Are customers to be notified?

Reply:
fixed. list commands match GA apis but they were not present in CLI before so no customer impact